openedx/features/colaraz_features/signals.py

```diff
@@ -0,0 +1,36 @@
+"""
+Signals for colaraz features django app.
+"""
+from logging import getLogger
+
+from django.conf import settings
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+
+from lms.djangoapps.verify_student.models import ManualVerification
+from student.models import UserProfile
+
+
+logger = getLogger(__name__)
+
+
+@receiver(post_save, sender=UserProfile)
+def generate_manual_verification_for_user(sender, instance, **kwargs):
+    """
+    Generate ManualVerification for the User (whose UserProfile instance has been created).
+    """
+    if not (settings.FEATURES.get('ENABLE_AUTOMATIC_ACCOUNT_VERIFICATION') and created):
+        if settings.FEATURES.get('ENABLE_AUTOMATIC_ACCOUNT_VERIFICATION'):
+            logger.info('ManualVerification for user: {} has already been created.'.format(instance.user.email))
+        return
+
+    logger.info('Generating ManualVerification for user: {}'.format(instance.user.email))
+    try:
+        ManualVerification.objects.create(
+            user=instance.user,
+            status='approved',
+            reason='SKIP_IDENTITY_VERIFICATION',
+            name=instance.name
+        )
```

Comment:
Add logging here instead, that Verification has been done for the user

Reply:
done.